src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const resultState = [];
```

Comment:
[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)